Core/Debugger/WebSocket/HLESubscriber.cpp

```diff
@@ -480,9 +493,42 @@ void WebSocketHLEFuncScan(DebuggerRequest &req) {
 	if (!req.ParamU32("size", &size))
 		return;
 
+	bool has_recreate = req.HasParam("recreate");
+	bool recreate = false;
+	if (has_recreate) {
+		if (!req.ParamBool("recreate", &recreate)) {
+			return;
+		}
+	}
+
 	if (!Memory::IsValidRange(addr, size))
 		return req.Fail("Address or size outside valid memory");
 
+	if (recreate) {
+		// The following code doesn't do what I wanted it to do AND it's debatable whether it's needed at all
+		// In short, I wanted to shift the start and the end of the range before removal
```

Comment:
I feel like it's best if the caller just has to look up and specify the full range.  Making functions overly magical just makes them confusing when you try to use them outside the narrow use case the API was originally designed thinking about.

Changing the range of the scan when remove/recreate/whatever is specified seems like it would be unexpected behavior for some.

-[Unknown]